### DISCUSSION TITLE
Merging 10x Genomics multiome Seurat objects

### DISCUSSION THREAD

**Discussion by jjwestfall521** [stuart-lab/signac#790] [Q&A] [answered]
Hello,

I have four datasets from the 10x Genomics mutiome (scRNA+scATAC) kit and have followed the vignette [Joint RNA and ATAC analysis: 10x multiomic](https://satijalab.org/signac/articles/pbmc_multiomic.html) for analyzing each sample separately. I would like to integrate these objects for some comparative analysis

 have a two-part question:

i. I have read the vignette [scATAC-seq data integration](https://satijalab.org/signac/articles/integrate_atac.html), but observed that one of the integrated objects lacks an RNA assay. My objects each have an RNA/SCT and ATAC assay. Can I integrate the objects using `SelectIntegrationFeatures()`, `FindIntegrationAnchors()` and `IntegrateData()` in the RNA/SCT assay if I am planning to use MACS2 for peak calling on the resulting integrated Seurat object?

ii. I have 4 different fragment files, one for each set. Is there a way to combine them to run MACS2?

Thanks for your help.

**Comment by timoast** [comment, marked as answer]
> i. I have read the vignette scATAC-seq data integration, but observed that one of the integrated objects lacks an RNA assay. My objects each have an RNA/SCT and ATAC assay. Can I integrate the objects using SelectIntegrationFeatures(), FindIntegrationAnchors() and IntegrateData() in the RNA/SCT assay if I am planning to use MACS2 for peak calling on the resulting integrated Seurat object?

Yes, following integration using the RNA or SCT assay you will have an object containing the integrated assay as well as merged assays for RNA / SCT, and the original ATAC assays. You can then set the default assay to ATAC to perform any analysis using the merged ATAC data.

> ii. I have 4 different fragment files, one for each set. Is there a way to combine them to run MACS2?

Yes, all fragment files present in the assay are used automatically if you run `CallPeaks()` on the merged assay, you do not need to combine the files yourself.
